### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by Bleu24** [community/community#143931] [GitHub Education] [answered]
### Hi everyone,

Hi, so I signed up for the Student Developer pack, and I've waited 3 days now, when can I specifically receive it?

**Comment by huddmax** [comment]
Hi! 👋 

The Student Developer Pack application process usually takes a few days, but in some cases, it might take up to a week for GitHub to review and approve your application. If you haven't heard back after a week, you could try reaching out to [GitHub Education Support](https://support.github.com/contact/education) for an update.

Hang in there; it’ll be worth it once it arrives! 😊

**Comment by davevad93** [comment, marked as answer]
Hi @Bleu24 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Bleu24 was approved and your benefits are now available.`

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
